### DISCUSSION TITLE
Pokédex terminada

### DISCUSSION THREAD

**Discussion by ErickMontesDK** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#450] [Show and tell]
Buenas tardes compañeros, le comparto mi proyecto.
Agradecería sus comentarios con el fin de mejorar.

Pagina web: https://erickmontesdk.github.io/Front-End-JS-Pokedex/

Repositorio: https://github.com/ErickMontesDK/Front-End-JS-Pokedex

**Comment by ValesApps** [comment]
esta genial q muestre las evoluciones y las formas :o

**Reply by RobertoVergaraD** [reply]
Me gusto mucho tu diseño, es mas o menos como me imagine que haría el mío, pero la verdad voy empezando en este mundo de la programación y me esta costando trabajo, pero vi tu código y como especificas todo me estas resolviendo muchísimas dudas!

Gracias por compartir tu repositorio!

**Reply by ErickMontesDK** [reply]
@RobertoVergaraD  Cualquier duda que tengas en mi código me puedes preguntar bro

**Comment by LuceroLuciano** [comment]
Te quedo Increíble, se ve que ya tienes bastante experiencia en esto del desarrollo web y con CSS puro, 100/100.

**Reply by ErickMontesDK** [reply]
Pues no, también ando empezando pero como estoy de momento de nini pues ando enfocado más a esto jaja. Gracias por tu comentario

**Comment by Vitaliquid** [comment]
Muchas gracias por compartir tu talento con la comunidad. La limpieza en el orden del código, las anotaciones, el detalle en la presentación de la información y missingno... un trabajo muy profesional que a los que no damos una nos brinda una idea mucho más clara de cómo se puede hacer una entrega profesional. me encantó y te estaré siguiendo. ¡Muchas gracias de nuevo!

**Comment by SujeiRodriguez** [comment]
esta bien genial :'D, que gran trabajo

**Comment by JonathanJRodriguez** [comment]
Hola Erick!

Me parece excelente tu pokedex, realmente creo que hiciste un gran trabajo en cuanto a presentación y funcionalidad.
Si aceptas puntos a mejorar aquí te dejo un par por si tienes tiempo después:
* Ponerle un efecto de presionar al botón de buscar, para retroalimentación al usuario.
*  Algunos tipos (electric por ejemplo) se sale el texto de la esferita.

Realmente se ve muy profesional, felicidades!
Saludos

**Reply by ErickMontesDK** [reply]
Gracias por el comentario. 
Si vi el detalle al buscar el tipo electrico pero no queria hacer más grande la sección. Igual ya lo acomodé.
Lo de la retroalimentación no lo habia notado, si hacia falta ahi. Ya le agregué un sonido al hacer la busqueda.
Saludos Jonathan

**Comment by MagdielAndres** [comment]
Se ve pro el pokedex
